armi/reactor/converters/uniformMesh.py

```diff
@@ -304,29 +302,19 @@ def _getFilteredFuelTopAndBottom(self):
         filteredTops : the top of fuel assemblies, filtered to a minimum separation of
             ``minimumMeshSize`` with preference for the top bounds
         """
-        assemblyTypeFlags = set(
-            [a.p.flags for a in self._sourceReactor.core.getAssemblies(Flags.FUEL)]
-        )
         fuelBottoms = [
-            min(
-                a.getFirstBlock(Flags.FUEL).p.zbottom
-                for a in self._sourceReactor.core.getAssemblies(assemFlags, exact=True)
-            )
-            for assemFlags in assemblyTypeFlags
+            a.getFirstBlock(Flags.FUEL).p.zbottom
+            for a in self._sourceReactor.core.getAssemblies(Flags.FUEL)
         ]
         fuelTops = [
-            max(
-                a.getBlocks(Flags.FUEL)[-1].p.ztop
-                for a in self._sourceReactor.core.getAssemblies(assemFlags, exact=True)
-            )
-            for assemFlags in assemblyTypeFlags
+            a.getBlocks(Flags.FUEL)[-1].p.ztop
+            for a in self._sourceReactor.core.getAssemblies(Flags.FUEL)
         ]
         filteredBottoms = self._filterMesh(
             fuelBottoms,
             self.minimumMeshSize,
             [min(fuelBottoms)],
             preference="bottom",
-            warn=True,
```

Comment:
Why do we want to remove this `warn=True` from here, but not from the determination of `filteredTops` below?